source/lex.tex

```diff
@@ -127,16 +127,16 @@
 source file to be processed from phase 1 through phase 4, recursively.
 All preprocessing directives are then deleted.
 
-\item Each basic source character set member in a character literal or a string
-literal, as well as each escape sequence and \grammarterm{universal-character-name} in a
-character literal or a non-raw string literal, is converted to the corresponding
+\item Each basic source character set member in a \grammarterm{character-literal} or a
+\grammarterm{string-literal}, as well as each escape sequence and \grammarterm{universal-character-name} in a
+\grammarterm{character-literal} or a non-raw string literal, is converted to the corresponding
 member of the execution character set~(\ref{lex.ccon}, \ref{lex.string}); if
 there is no corresponding member, it is converted to an \impldef{converting
 characters from source character set to execution character set} member other
 than the null (wide) character.\footnote{An implementation need not convert all
 non-corresponding source characters to the same execution character.}
 
-\item Adjacent string literal tokens are concatenated.
+\item Adjacent \grammarterm{string-literal} tokens are concatenated.
```

Comment:
This is not correct; as specified in [lex.ext]p8, this also includes *user-defined-string-literal* tokens, and we're definitely talking about complete tokens and not lexer rules here.

Reply:
Reverted.